pkg/cli/alpha/internal/generate.go

```diff
@@ -392,3 +392,44 @@ func kubebuilderGrafanaEdit() error {
 	}
 	return nil
 }
+
+func RunGenerate(opts *Generate) error {
```

Comment:
Why it was added in the internal?
Did you copy the code from `if err := opts.Generate(); ?`

Reply:
You replaced `opts.Generate();` with this method right?
Why you need to do this change
Have we now the implementation in opts and here? 
Are not both duplicated?

Reply:
yes both are duplicated so do we need to keep the same `opts.Generate();` instead of `internal.RunGenerate(&opts);`?